manifest.json

```diff
@@ -16,7 +16,7 @@
     "content_scripts": [
     {
         "matches": ["http://*/*", "https://*/*"],
-        "run_at": "document_start",
+        "run_at": "document_end",
```

Comment:
This makes the detection significantly slower for finding clerk, and increases the possibility it wont run at all if the page has other js errors.